editor/code_editor.cpp

```diff
@@ -1683,10 +1683,6 @@ void CodeTextEditor::goto_error() {
 }
 
 void CodeTextEditor::_update_text_editor_theme() {
-	if (!EditorThemeManager::is_generated_theme_outdated()) {
```

Comment:
Won't the removal of this check cause a whole slew of unnecessary updating?

Reply:
This seems to be called only from `NOTIFICATION_THEME_CHANGED`, so the condition was wrongly added .-. (I blame the rebase)

Reply:
Thus is what happens with last-minute rebases. A beautiful experience.